docs/health/health_design/design.rst

```diff
@@ -119,19 +177,19 @@ However, you can select which testcase and which sections to execute pyATS Healt
 
 There are three ways to filter it. By default pyATS Health Checks run before and after every testcase and section. With the filtering you can decide where they are executed.
 
-Testcase level: health-uids, provide the testcase/trigger names from Testcase/Trigger datafile. The exact name can be provided or regular expression is also supported. pyATS Health Check processors will run only for the given testcase/trigger names which match the full name or match the regex.
+Testcase level: health-tc-uids, provide the testcase/trigger names from Testcase/Trigger datafile. The exact name can be provided or regular expression is also supported. pyATS Health Check processors will run only for the given testcase/trigger names which match the full name or match the regex.
```

Comment:
why has `-tc` been added? 'common_setup' / 'CommonSetup' is technically not a testcase but it can still have a processor attached to it

Reply:
It was requested to change internally. still old name `health-sections`/`health-uids` works as deprecated. I will take a note to discuss. `-tc` might be confusing. but no functionality issue even against `common_setup/`CommonSetup`